pyproject.toml

```diff
@@ -24,23 +24,27 @@ classifiers = [
 
 [tool.poetry.dependencies]
 python = "^3.10.0"
-bento-lib = {extras = ["flask"], version = "^13.5.0"}
+bento-lib = {extras = ["fastapi"], version = "^13.5.0"}
 celery = {extras = ["redis"], version = "^5.5.3"}
 requests = "^2.32.4"
-Flask = "~3.1.1"
 pydantic = "^2.7.1"
-flask-cors = "^6.0.0"
 werkzeug = "~3.1.3"
 asgiref = "^3.8.1"
+fastapi = "^0.116.1"
+uvicorn = "^0.35.0"
+httpx = "^0.28.1"
+a2wsgi = "^1.10.10"
```

Comment:
q: what is this for?

Reply:
I was using it during the initial dev, removed it now